modules/apps/commerce/commerce-product-test/src/testFunctional/macros/json/CommerceJSONProductsAPI.macro

```diff
@@ -270,6 +270,60 @@ definition {
 		com.liferay.poshi.runner.util.JSONCurlUtil.post("${curl}");
 	}
 
+	macro _AddCommerceProductImage {
```

Comment:
The JSON macro names start in lower case, then continue with camel case, please see the other macros and edit this one: `_AddCommerceProductImage`  to `_addCommerceProductImage`

Reply:
Fixed